TAO/tests/Oneway_Send_Timeouts/Server_Task.h

```diff
@@ -28,7 +28,7 @@ class Server_Task : public ACE_Task_Base
   ACE_ARGV my_args (args_.c_str());
 
   // Initialize Server ORB in new thread
-  ACE_auto_ptr_reset (server_, new Server (my_args.argc(), my_args.argv()));
+  server_ = std::make_unique<Server>(my_args.argc(), my_args.argv());
```

Comment:
`std::make_unique` is a C++14 feature so you can't use it as much as we would like to do that.

Reply:
You are right, but it builds on CI?

Reply:
CI doesn't build all, a full build probably takes up to 10 hours

Reply:
OK